model/clusters_mgmt/v1/provision_shard_type.model

```diff
@@ -38,9 +38,12 @@ class ProvisionShard {
 	// Contains the cloud provider name
 	link CloudProvider CloudProvider
 
-	// Contains the name of the management cluster for Hypershift clusters that are assigned to this shard.
+	// Contains the configuration for Hypershift
+	HypershiftConfig ServerConfig
```

Comment:
comment should end with `.` 

Also the one above (cloud provider name), which is not part of this PR...

Reply:
non of the comments in this file ends with a dot.

Reply:
Fixed. PreTAL

Reply:
Is this the ServerConfig of the Service Cluster? HypershiftConfig is a bit confusing as it suggest that this is the config of the hosted hypershift cluster. Something along the line ServiceClusterConfig might be better.

Reply:
Given that CS is cluster service, naming something "service cluster" is very confusing IMO.

From OCM perspective, there are two types of provisioning shards - hive, and hypershift.

That's also the name of the field in CS model and DB.